feature/utils.py

```diff
@@ -78,6 +78,36 @@ def normalize_columns(df: pd.DataFrame):
     return df_normalized.fillna(0)
 
 
+def _process_df(input_df: pd.DataFrame, categories: List[str], feature_column: str, selection_size: int) \
```

Comment:
what's the difference between this process_df function and the other process_df function?

Reply:
do we need both?

Reply:
No, we do not need both. I explained in a few lines at the tope of _process_df function in text_based.py

Reply:
remind me again what's the difference between this process and the other one text_based..